_data/resources.yml

```diff
@@ -217,4 +223,8 @@
       title: "Sass a11y"
     - url: http://tenon.io/
       title: "Tenon.io Accessibility Testing API"
+    - url: http://jxnblk.com/colorable/demos/text/
```

Comment:
Are these added twice? Let's try not duplicating and maybe just choose category.